converters/fromZigbee.js

```diff
@@ -3316,6 +3316,60 @@ const converters = {
             return payload;
         },
     },
+    humidity2: {
+        cluster: 'msRelativeHumidity',
+        type: ['attributeReport', 'readResponse'],
+        options: [exposes.options.precision('humidity'), exposes.options.calibration('humidity')],
+        convert: (model, msg, publish, options, meta) => {
```

Comment:
Please integrate these changes into the existing converters.

Reply:
@Koenkk, I am  agree that if a device supports multiEndpoint, it should always append a suffix. But this update may break integration with HA or other systems for many devices. Therefore, I have added a separate converter.

Reply:
I see, please move it into fzLocal of `custom_devices_diy.js`

Reply:
@Koenkk Maybe, add it to fromZigbee.js and start moving to the correct implementation for new devices?

Reply:
I want to go for a diff solution in the future, so lets move it to `custom_devices_diy.js` for now.